azure/requests.yaml

```
@@ -34,3 +24,7 @@ releases:
           issue: https://github.com/giantswarm/giantswarm/issues/13671
         - name: kubernetes
           version: ">= 1.17.12"
+        - name: app-operator
+          version: ">=2.3.2"
+        - name: chart-operator
+          version: ">=2.3.3"
```

```suggestion
          version: ">=2.3.5"
```

Thanks Marta! Sorry to be a pain but could you include the chart-operator v2.3.5? It's a bug fix release we created yesterday.

https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#235---2020-10-13

---

sure, we're not testing the release yet anyway